Source/WebCore/platform/graphics/coretext/FontCustomPlatformDataCoreText.cpp

```diff
@@ -88,15 +89,37 @@ RefPtr<FontCustomPlatformData> createFontCustomPlatformData(SharedBuffer& buffer
 
     // Retain the extracted font contents, so the GPU process doesn't have to extract it a second time later.
     // This is a power optimization.
-    auto extractedData = adoptCF(FPFontCopySFNTData(font));
+    return adoptCF(FPFontCopySFNTData(font));
+}
+
+RefPtr<FontCustomPlatformData> FontCustomPlatformData::create(SharedBuffer& buffer, const String& itemInCollection)
+{
+    auto extractedData = extractFontCustomPlatformData(buffer, itemInCollection);
```

Comment:
`auto` -> `RetainPtr`. We prefer not to hide smart pointers behind `auto` in new code.